crates/optimism/node/src/node.rs

```diff
@@ -196,8 +196,15 @@ where
     }
 
     fn add_ons(&self) -> Self::AddOns {
+        let sequencer = self
+            .args
+            .sequencer
+            .as_ref()
+            .or(self.args.sequencer_http.as_ref())
+            .or(self.args.sequencer_ws.as_ref())
+            .cloned();
```

Comment:
I'd like to have this as standalone function in case rollupargs is used somewhere else